### ISSUE TITLE
Harmonizing Identity-Related Web Platform APIs

### ISSUE THREAD

**Issue by timcappalli** [w3c/tpac2024-breakouts#49]
### Session description

There are currently three active work streams for identity-related Web Platform APIs in the W3C: the [Digital Credentials API (DC API)](https://wicg.github.io/digital-credentials/), [Federated Credential Management (FedCM)](https://w3c-fedid.github.io/FedCM/), and the [Web Authentication API (WebAuthn)](https://w3c.github.io/webauthn/).

In each of these work streams, there have been some common discussion items including flexible but privacy preserving error codes, selector experience and query syntax, conditionally mediated flows, along with discussion around how these APIs should interact in the future (ex: requesting a VDC -or- an OIDC ID Token in one call).

This breakout is intended to be a collaborative, working session. The focus will be on gaining consensus on the more tactical items like errors codes and selectors, with some time at the end reserved for forward looking ideas.

### Session goal

This breakout is intended to be a collaborative, working session. The focus will be on gaining consensus on the more tactical items like errors codes and selectors, with some time at the end reserved for forward looking ideas.

### Additional session chairs (Optional)

@samuelgoto @marcoscaceres @masterkale

### Who can attend

Anyone may attend (Default)

### Other sessions where we should avoid scheduling conflicts (Optional)

#10 #12 #16 #24

**Comment by ianbjacobs** [comment]
@timcappalli, is this session intended as follow up to the session we are discussing for Monday afternoon?

**Comment by timcappalli** [comment]
No, I'd like to keep payments out of scope for this one as it's being discussed in other sessions.

**Comment by ianbjacobs** [comment]
Would you mind updating the description to clarify that the scope is not intended to include payments, and we intend to hold more payments-focused discussion on Monday afternoon within the Web Payments WG meeting?

**Comment by timcappalli** [comment]
Sure. I highlighted the topics to be discussed in the description, but I can add an explicit mention of payments.
